src/content/service.js

```diff
@@ -15,53 +16,94 @@ const getShortUid = () =>
     .join('')
     .substr(0, 6)
 
-module.exports = ({ db, botfile, projectLocation, logger }) => {
-  let categories = []
+const getNewItemId = category => {
+  const prefix = (category.ummBloc || category.id).replace(/^#/, '')
+  return `${prefix}-${getShortUid()}`
+}
 
-  const scanAndRegisterCategories = async () => {
-    categories = []
+const prepareDb = async () => {
+  const knex = getInMemoryDb()
+
+  await knex.schema.createTable('content_items', table => {
```

Comment:
I think [we should use this](https://github.com/botpress/botpress/blob/master/src/database/helpers.js#L44) as if the table exists it will crash?

Reply:
ok just saw it's an in memory session but let's use it anyway for consistency purposes